src/Slack.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Behatch\Notifier;
+
+class Slack extends Notifier
+{
+    static private $url;
+	static private $settings;
+	static private $prefix;
+	static private $attachment;
+
+    public function __construct($url, $settings = [], $prefix = null, $attachment = [], $spamTimeout = 60)
+    {
+        parent::__construct($spamTimeout);
+
+        self::$url = $url;
+        self::$prefix = $prefix;
+
+        self::$settings = $settings + [
+            'username' => 'Behat',
+            'channel' => '#general',
+            'link_names' => true,
+			'icon' => ':fire:',
+        ];
+
+		self::$attachment = $attachment + [
+			'fallback' => 'Behat test failed',
+			'text' => '',
+			'pretext' => '',
+			'color' => 'danger',
+			'fields' => array(),
+		];
+    }
+
+    protected static function notify($status, $title, $message)
+    {
+
+		// We don't need the second notification about the suite failing
+		if (strpos($title, 'suite ended') !== false) {
+			return;
+		}
+
+		// If there's a prefix, prepend it to the title of the message
+        if (self::$prefix !== null) {
+            $title = '[' . self::$prefix . '] ' . $title;
+        }
+
+		// Fire up the Slack Client with the given webhook url and settings
+		$client = new \Maknz\Slack\Client(self::$url, self::$settings);
```

Comment:
To prevent an horrible backtrace, check in the constructor if this class exist and print a comprehensible message for the user.